examples/snippets/chart-js-rs-e417185604eb4a58/inline0.js

```diff
@@ -0,0 +1,16 @@
+export function render_chart(v, id, mutate) {
```

Comment:
before committing, please delete the snippets folder and rerun build_example once. Should keep this folder clean